nano/core_test/bootstrap.cpp

```diff
@@ -400,26 +394,33 @@ TEST (bootstrap_processor, process_new)
 	config.frontiers_confirmation = nano::frontiers_confirmation_mode::disabled;
 	nano::node_flags node_flags;
 	node_flags.disable_bootstrap_bulk_push_client = true;
-	auto node1 (system.add_node (config, node_flags));
+	nano::keypair key2;
+
+	auto node1 = system.add_node (config, node_flags);
 	config.peering_port = system.get_available_port ();
-	auto node2 (system.add_node (config, node_flags));
+	auto node2 = system.add_node (config, node_flags);
+
 	system.wallet (0)->insert_adhoc (nano::dev::genesis_key.prv);
-	nano::keypair key2;
 	system.wallet (1)->insert_adhoc (key2.prv);
-	auto send (system.wallet (0)->send_action (nano::dev::genesis_key.pub, key2.pub, node1->config.receive_minimum.number ()));
+
+	// send amount raw from genesis to key2, the wallet will autoreceive
+	auto amount = node1->config.receive_minimum.number ();
+	auto send = system.wallet (0)->send_action (nano::dev::genesis_key.pub, key2.pub, amount);
 	ASSERT_NE (nullptr, send);
-	ASSERT_TIMELY (10s, !node1->balance (key2.pub).is_zero ());
-	auto receive (node2->block (node2->latest (key2.pub)));
+	ASSERT_TIMELY (5s, !node1->balance (key2.pub).is_zero ());
+	auto receive = node2->block (node2->latest (key2.pub));
 	ASSERT_NE (nullptr, receive);
```

Comment:
There might have been an existing issue here. I don't see how there would be a guarantee that node2 has received the block that was generated on node1 immediately.